zipline/data/data_portal.py

```diff
@@ -311,6 +323,146 @@ def _get_daily_data(self, asset_int, column, dt):
         else:
             return raw_value
 
+    def _get_history_daily_window(self, sids, end_dt, bar_count, field_to_use):
+        """
+        Internal method that returns a dataframe containing history bars
+        of daily frequency for the given sids.
+        """
+        data = []
```

Comment:
I think we could use the FFC (now moved to zipline.data.us_equity_pricing) USEquityPricingLoader load_adjusted_array to get same result?

Is there particular logic that is unique to this window?